modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/containers/Form.es.js

```diff
@@ -122,59 +133,73 @@ const Form = React.forwardRef((props, ref) => {
 				portletNamespace,
 				rules,
 				successPageSettings,
-			};
-		},
-		validate,
-	}));
-
-	useEffect(() => {
-		if (containerRef.current) {
-			Liferay.fire('ddmFormPageShow', {
-				formId: getFormId(getFormNode(containerRef.current)),
-				page: props.activePage,
-				title: props.pages[props.activePage].title,
-			});
-		}
-		// eslint-disable-next-line react-hooks/exhaustive-deps
-	}, [containerRef]);
-
-	useEffect(() => {
-		let domDelegatedEventHandle = null;
-		const form = null;
-
-		if (containerRef.current) {
-			const form = getFormNode(containerRef.current);
-
-			if (form) {
-				domDelegatedEventHandle = dom.on(
-					form,
-					'submit',
-					handleFormSubmitted
-				);
+			}),
+			validate,
+		}));
+
+		useEffect(() => {
+			if (containerRef.current) {
+				Liferay.fire('ddmFormPageShow', {
+					formId: getFormId(getFormNode(containerRef.current)),
+					page: activePage,
+					title: pages[activePage].title,
+				});
 			}
+			// eslint-disable-next-line react-hooks/exhaustive-deps
+		}, []);
 
-			Liferay.on(
-				'submitForm',
-				handleLiferayFormSubmitted.bind({form}),
-				this
-			);
-		}
-
-		return () => {
-			Liferay.detach(
-				'submitForm',
-				handleLiferayFormSubmitted.bind({form}),
-				this
-			);
-
-			if (domDelegatedEventHandle) {
-				domDelegatedEventHandle.removeListener();
+		useEffect(() => {
+			let submitHandle;
+
+			let onHandle;
```

Comment:
There's a small chance that Brian might reject this based on the order, but it's a tiny cosmetic issue so I think we should try forwarding anyway. Can always fix it next time we're in this file.

Reply:
🥺 is it necessary to still have a line? I was wondering if in this case, it was necessary haha, I ended up adding.

Reply:
Technically, I don't think you need it but I kept it anyway because I wanted to keep the change as minimal as possible...

It might be that there are some subtle rules that Brian has which I don't know yet — I tried to write up his logic here:

https://github.com/liferay/eslint-config-liferay/issues/160

Reply:
ok, thanks! It still seems a little confusing 😅, I’ll reread it a few more times.